src/borg/archive.py

```diff
@@ -1459,7 +1468,7 @@ def identify_key(self, repository):
         cdata = repository.get(some_chunkid)
         return key_factory(repository, cdata)
 
-    def verify_data(self):
+    def do_verify_data(self):
```

Comment:
why did you rename this?

Reply:
as we now have a boolean flag we need to save to `self`:

```
def __init__(self, ..., verify_data=False):
    ...
    self.verify_data = verify_data
```